### PR TITLE
Fix default compress algo

### PR DESCRIPTION
**Pull request by yuyicg**
change the default compression algorithm in writer.py from gzip to snappy.

### CODE REVIEW

**Review by skydoorkai — APPROVED** [review]
Compressor(2) 最好用定义个Header.Snappy = 2, Compressor(Header.Snappy)比较可读。

File: recordio/recordio/file.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 from recordio.recordio.reader import Reader
+from recordio.recordio.header import Compressor
 
 
 class File(object):
     """ Simple Wrapper for FileIndex, Writer and Reader for usability.
     """
 
-    def __init__(self, file_path, mode, *, max_chunk_size=1024):
+    def __init__(self, file_path, mode, *, max_chunk_size=1024, compressor=Compressor(2)):
```
Comment (zou000) [inline recordio/recordio/file.py]: 可直接用compresser=Compressor.snappy

File: recordio/recordio/writer.py [hunk 2]
```diff
@@ -7,7 +7,7 @@ class Writer(object):
     """ Util class for user to transfer raw data into recordio chunk format.
     """
 
-    def __init__(self, out_file, max_chunk_size, compressor=Compressor(3)):
+    def __init__(self, out_file, max_chunk_size, compressor=Compressor(2)):
```
Comment (zou000) [inline recordio/recordio/writer.py]: same

File: recordio/tensorflow_op/cc/recordio_chunk_parser.cc [hunk 3]
```diff
@@ -26,12 +26,12 @@ Status ReadNBytes(InputStreamInterface* input_stream,
...
   // TODO
-  return errors::Unimplemented("");
+  return errors::Unimplemented("gzip decompression not support for now.");
 }
 
 Status ZlibDecompress(const string* src, string* dst) {
   // TODO
-  return errors::Unimplemented("");
+  return errors::Unimplemented("zlib decompression not support for now.");
 }
```
Comment (zou000) [inline recordio/tensorflow_op/cc/recordio_chunk_parser.cc:36]: code更改后需重新build artifacts下面的包

**Review by zou000 — CHANGES_REQUESTED** [review]

**Review by zou000 — APPROVED** [review]

**Pull request merged**